prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -208,11 +217,19 @@ elements:
       documentation: This is the coded value that describes the result. For IgG, IgM and CT results that provide a value leave this field blank.
 
     # column AT
-    - name: testing_lab_name
+    # nonstandard. need to concat because we currently can't add the same column twice to a table
+    # TODO: fix this
+    - name: name_of_testing_lab
+      type: TEXT
+      mapper: 'concat(testing_lab_name)'
```

Comment:
I don't understand what's the requirement here?  To show the same lab name string under two different column headers?  That would be 
`csvFields: [{ name: Foo}, {name: Bar}]` 
?

Reply:
change to local_testing_lab_name.
Use use(testing_lab_name)